src/routes/stake/Stake.svelte

```diff
@@ -133,11 +133,8 @@
 		<div class="reward">
 			<p style:margin-right={'2.5em'}>
 				Future WTN Airdrop:
-				{#if exchangeRate && totalIcpDeposited}
-					{computeRewards(
-						totalIcpDeposited,
-						computeReceiveAmount(true, BigNumber($inputAmount), exchangeRate)
-					).toFixed(2)}
+				{#if totalIcpDeposited && !BigNumber($inputAmount).isNaN()}
+					{displayUsFormat(computeRewards(totalIcpDeposited, BigNumber($inputAmount)), 8)}
```

Comment:
8 decimals for a number that can have 3 decimals at most?

Reply:
not necessary that's true however it will not display 8 decimals for nothing but just up to 8 decimals.